tests/unit/services/test_predictor_service.py

```
@@ -0,0 +1,96 @@
+        time_array = np.array([0, 1])
+        value_array = np.array([50, 55])
+        predictor = FuturePredictorService(mock_equation, mock_settings)
+        with pytest.raises(ValueError):
+            predictor.predict(time_array, value_array)
+
+    @patch("os.makedirs")
+    @patch("pandas.DataFrame.to_excel")
+    def test_save_prediction_to_excel(
+        self, mock_to_excel, mock_makedirs, mock_equation, mock_settings
+    ):
+        """予測結果が正しくExcelに保存されるか（の呼び出しが行われるか）テスト"""
```

Ruff (0.12.2)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)